### ISSUE TITLE
Italian localisation

### ISSUE THREAD

**Issue by lzambarda** [kohii/smoothcsv3#6]
Hello there, congrats on the new release! I have read that SmoothCSV now supports localisation and I wanted to let you know that I'd be keen in providing Italian localisation.

**Comment by kohii** [comment]
Thank you for your interest!

I should mention that the existing Spanish (es) and Chinese (zh-cn) translations were initially generated using AI.

While I could create Italian translations the same way, I'm not sure if the translations are natural.
It would be very helpful if you could review the Italian translations.

If you're interested in helping, I can share the JSON localization files here for your review. SmoothCSV uses a localization system similar to [VSCode's approach](https://github.com/microsoft/vscode-l10n), with all translated strings stored in JSON files.

Please let me know if you'd like to help, and I'll upload the Italian translation files.

**Comment by lzambarda** [comment]
Sure, I am happy to help providing a good translation! 👍

Side note: I never used AI to do translation, it would be nice to see how it compares.

**Comment by kohii** [comment]
Thank you for your enthusiasm to help!

Here are the Italian translations for your review:

## `package.nls.json` and `package.nls.it.json`

`package.nls.json` is for commands and menu items.
Commands are shown in the command palette and menu.

This is the English original:

`package.nls.json`

```package.nls.json
{
	"commandCategory.cellEditor": "Cell Editor",
	"commandCategory.gridEditor": "Grid Editor",
	"commandCategory.file": "File",
	"commandCategory.view": "View",
	"commandCategory.help": "Help",

	"command.app.quit": "Quit",
	"command.app.checkForUpdate": "Check for Update...",
	"command.app.installUpdateAndRestart": "Restart to Update",
	"command.app.configureDisplayLanguage": "Configure Display Language",

	"command.copy": "Copy",
	"command.cut": "Cut",
	"command.paste": "Paste",
	"command.selectAll": "Select All",
	"command.undo": "Undo",
	"command.redo": "Redo",

	"command.cellEditor.insertNewLine": "Insert New Line",
	"command.cellEditor.startEditingCell": "Start Editing Cell",
	"command.cellEditor.toggleEditMode": "Toggle Edit Mode and Enter Mode",
	"command.cellEditor.finishEditingCell": "Finish Editing Cell",
	"command.cellEditor.cancelEditingCell": "Cancel Editing Cell",
	"command.gridEditor.clear": "Clear",
	"command.gridEditor.insertRowsAbove": "Insert Rows Above",
	"command.gridEditor.insertRowsBelow": "Insert Rows Below",
	"command.gridEditor.insertColumnsLeft": "Insert Columns Left",
	"command.gridEditor.insertColumnsRight": "Insert Columns Right",
	"command.gridEditor.deleteRows": "Delete Rows",
	"command.gridEditor.deleteColumns": "Delete Columns",
	"command.gridEditor.deleteCells": "Delete Cells",
	"command.gridEditor.insertCells": "Insert Blank Cells",
	"command.gridEditor.trimContent": "Remove Trailing Blank Rows and Columns",
	"command.gridEditor.fillDown": "Fill Down",
	"command.gridEditor.fillUp": "Fill Up",
	"command.gridEditor.fillRight": "Fill Right",
	"command.gridEditor.fillLeft": "Fill Left",
	"command.gridEditor.autoFitColumns": "Auto-fit Column Widths",
	"command.gridEditor.changeFileFormat": "Change File Format...",
	"command.gridEditor.setHeaderRows": "Set Header Rows...",
	"command.gridEditor.setHeaderColumns": "Set Header Columns...",
	"command.gridEditor.setHeaderRowsUpToSelection": "Set Header Rows up to Selection",
	"command.gridEditor.setHeaderColumnsUpToSelection": "Set Header Columns up to Selection",
	"command.gridEditor.setHeaderRowsAndColumnsUpToSelection": "Set Header Rows and Columns up to Selected Cell",
	"command.gridEditor.unsetHeaderRows": "Unset Header Rows",
	"command.gridEditor.unsetHeaderColumns": "Unset Header Columns",
	"command.gridEditor.unsetHeaders": "Unset Headers",
	"command.gridEditor.copyUsingFileFormat": "Copy Using File Format",
	"command.gridEditor.cutUsingFileFormat": "Cut Using File Format",
	"command.gridEditor.pasteUsingFileFormat": "Paste Using File Format",
	"command.gridEditor.copyAsMarkdown": "Copy as Markdown Table",
	"command.gridEditor.pasteAsMarkdown": "Paste as Markdown Table",
	"command.gridEditor.goToRow": "Go to Row/Column...",
	"command.gridEditor.find": "Find",
	"command.gridEditor.replace": "Replace",
	"command.gridEditor.closeFindWidget": "Close Find Widget",
	"command.gridEditor.findNext": "Find Next",
	"command.gridEditor.findPrevious": "Find Previous",
	"command.gridEditor.replaceNext": "Replace Next",
	"command.gridEditor.replaceAll": "Replace All",
	"command.gridEditor.sort": "Sort Rows...",
	"command.gridEditor.sort.shortTitle": "Advanced Sort...",
	"command.gridEditor.sortCells": "Sort Selected Cells...",
	"command.gridEditor.sortRows": "Sort Selected Rows...",
	"command.gridEditor.sort.aToZ": "Sort A to Z",
	"command.gridEditor.sort.zToA": "Sort Z to A",
	"command.gridEditor.sort.smallestToLargest": "Sort Smallest to Largest",
	"command.gridEditor.sort.largestToSmallest": "Sort Largest to Smallest",
	"command.gridEditor.sort.oldestToNewest": "Sort Oldest to Newest",
	"command.gridEditor.sort.newestToOldest": "Sort Newest to Oldest",
	"command.gridEditor.sort.shortestToLongest": "Sort Shortest to Longest",
	"command.gridEditor.sort.longestToShortest": "Sort Longest to Shortest",
	"command.gridEditor.transform.toUpperCase": "Transform to Uppercase",
	"command.gridEditor.transform.toLowerCase": "Transform to Lowercase",
	"command.gridEditor.transform.swapCase": "Swap Case",
	"command.gridEditor.transform.toCamelCase": "Transform to Camel Case",
	"command.gridEditor.transform.toSnakeCase": "Transform to Snake Case",
	"command.gridEditor.transform.toPascalCase": "Transform to Pascal Case",
	"command.gridEditor.transform.toTitleCase": "Transform to Title Case",
	"command.gridEditor.transform.toKebabCase": "Transform to Kebab Case",

	"command.gridEditor.selection.selectRow": "Select Current Row",
	"command.gridEditor.selection.selectColumn": "Select Current Column",

	"command.showAllCommands": "Show All Commands",
	"command.showAllCommands.shortTitle": "Command Palette",

	"command.file.newUntitledFile": "New File",
	"command.file.save": "Save",
	"command.file.saveAs": "Save As...",
	"command.file.saveAll": "Save All",
	"command.file.open": "Open...",
	"command.file.reload": "Reload from Disk",
	"command.file.reloadWithAutoDetect": "Reload with Auto-Detect File Format",
	"command.file.copyFilePath": "Copy Path of Active File",
	"command.file.copyFilePath.shortTitle": "Copy Path",

	"command.view.closeEditor": "Close Editor",
	"command.view.closeEditor.shortTitle": "Close",
	"command.view.closeAllEditors": "Close All Editors",
	"command.view.closeAllEditors.shortTitle": "Close All",
	"command.view.closeOtherEditors": "Close Other Editors",
	"command.view.closeOtherEditors.shortTitle": "Close Others",
	"command.view.focusNextEditor": "Focus Next Editor",
	"command.view.focusPreviousEditor": "Focus Previous Editor",

	"command.help.reportIssue": "Report Issue",

	"menu.file": "File",
	"menu.edit": "Edit",
	"menu.grid": "Grid",
	"menu.view": "View",
	"menu.window": "Window",
	"menu.help": "Help",

	"menu.copyAs": "Copy As",
	"menu.pasteAs": "Paste As",
	"menu.headers": "Headers",
	"menu.fill": "Fill",
	"menu.grid.sort": "Sort",
	"menu.transform": "Transform",
	"menu.setHeaderRows1": "Set Header Rows to 1",
	"menu.setHeaderRows2": "Set Header Rows to 2",
	"menu.setHeaderColumns1": "Set Header Columns to 1",
	"menu.setHeaderColumns2": "Set Header Columns to 2"
}
```

This is AI-generated Italian translation for `package.nls.json`:

`package.nls.it.json`

```package.nls.it.json
{
	"commandCategory.cellEditor": "Editor Celle",
	"commandCategory.gridEditor": "Editor Griglia",
	"commandCategory.file": "File",
	"commandCategory.view": "Visualizza",
	"commandCategory.help": "Aiuto",

	"command.app.quit": "Esci",
	"command.app.checkForUpdate": "Controlla aggiornamenti...",
	"command.app.installUpdateAndRestart": "Riavvia per aggiornare",
	"command.app.configureDisplayLanguage": "Configura lingua di visualizzazione",

	"command.copy": "Copia",
	"command.cut": "Taglia",
	"command.paste": "Incolla",
	"command.selectAll": "Seleziona tutto",
	"command.undo": "Annulla",
	"command.redo": "Ripeti",

	"command.cellEditor.insertNewLine": "Inserisci nuova riga",
	"command.cellEditor.startEditingCell": "Inizia modifica cella",
	"command.cellEditor.toggleEditMode": "Attiva/disattiva modalità modifica",
	"command.cellEditor.finishEditingCell": "Termina modifica cella",
	"command.cellEditor.cancelEditingCell": "Annulla modifica cella",
	"command.gridEditor.clear": "Cancella",
	"command.gridEditor.insertRowsAbove": "Inserisci righe sopra",
	"command.gridEditor.insertRowsBelow": "Inserisci righe sotto",
	"command.gridEditor.insertColumnsLeft": "Inserisci colonne a sinistra",
	"command.gridEditor.insertColumnsRight": "Inserisci colonne a destra",
	"command.gridEditor.deleteRows": "Elimina righe",
	"command.gridEditor.deleteColumns": "Elimina colonne",
	"command.gridEditor.deleteCells": "Elimina celle",
	"command.gridEditor.insertCells": "Inserisci celle vuote",
	"command.gridEditor.trimContent": "Rimuovi righe e colonne vuote finali",
	"command.gridEditor.fillDown": "Riempi in basso",
	"command.gridEditor.fillUp": "Riempi in alto",
	"command.gridEditor.fillRight": "Riempi a destra",
	"command.gridEditor.fillLeft": "Riempi a sinistra",
	"command.gridEditor.autoFitColumns": "Adatta larghezza colonne",
	"command.gridEditor.changeFileFormat": "Cambia formato file...",
	"command.gridEditor.setHeaderRows": "Imposta righe intestazione...",
	"command.gridEditor.setHeaderColumns": "Imposta colonne intestazione...",
	"command.gridEditor.setHeaderRowsUpToSelection": "Imposta righe intestazione fino alla selezione",
	"command.gridEditor.setHeaderColumnsUpToSelection": "Imposta colonne intestazione fino alla selezione",
	"command.gridEditor.setHeaderRowsAndColumnsUpToSelection": "Imposta righe e colonne intestazione fino alla cella selezionata",
	"command.gridEditor.unsetHeaderRows": "Rimuovi righe intestazione",
	"command.gridEditor.unsetHeaderColumns": "Rimuovi colonne intestazione",
	"command.gridEditor.unsetHeaders": "Rimuovi intestazioni",
	"command.gridEditor.copyUsingFileFormat": "Copia usando formato file",
	"command.gridEditor.cutUsingFileFormat": "Taglia usando formato file",
	"command.gridEditor.pasteUsingFileFormat": "Incolla usando formato file",
	"command.gridEditor.copyAsMarkdown": "Copia come tabella Markdown",
	"command.gridEditor.pasteAsMarkdown": "Incolla come tabella Markdown",
	"command.gridEditor.goToRow": "Vai a riga/colonna...",
	"command.gridEditor.find": "Trova",
	"command.gridEditor.replace": "Sostituisci",
	"command.gridEditor.closeFindWidget": "Chiudi widget di ricerca",
	"command.gridEditor.findNext": "Trova successivo",
	"command.gridEditor.findPrevious": "Trova precedente",
	"command.gridEditor.replaceNext": "Sostituisci successivo",
	"command.gridEditor.replaceAll": "Sostituisci tutto",
	"command.gridEditor.sort": "Ordina righe...",
	"command.gridEditor.sort.shortTitle": "Ordinamento avanzato...",
	"command.gridEditor.sortCells": "Ordina celle selezionate...",
	"command.gridEditor.sortRows": "Ordina righe selezionate...",
	"command.gridEditor.sort.aToZ": "Ordina da A a Z",
	"command.gridEditor.sort.zToA": "Ordina da Z a A",
	"command.gridEditor.sort.smallestToLargest": "Ordina dal più piccolo al più grande",
	"command.gridEditor.sort.largestToSmallest": "Ordina dal più grande al più piccolo",
	"command.gridEditor.sort.oldestToNewest": "Ordina dal più vecchio al più recente",
	"command.gridEditor.sort.newestToOldest": "Ordina dal più recente al più vecchio",
	"command.gridEditor.sort.shortestToLongest": "Ordina dal più corto al più lungo",
	"command.gridEditor.sort.longestToShortest": "Ordina dal più lungo al più corto",
	"command.gridEditor.transform.toUpperCase": "Trasforma in maiuscolo",
	"command.gridEditor.transform.toLowerCase": "Trasforma in minuscolo",
	"command.gridEditor.transform.swapCase": "Inverti maiuscole/minuscole",
	"command.gridEditor.transform.toCamelCase": "Trasforma in camel case",
	"command.gridEditor.transform.toSnakeCase": "Trasforma in snake case",
	"command.gridEditor.transform.toPascalCase": "Trasforma in pascal case",
	"command.gridEditor.transform.toTitleCase": "Trasforma in title case",
	"command.gridEditor.transform.toKebabCase": "Trasforma in kebab case",

	"command.gridEditor.selection.selectRow": "Seleziona riga corrente",
	"command.gridEditor.selection.selectColumn": "Seleziona colonna corrente",

	"command.showAllCommands": "Mostra tutti i comandi",
	"command.showAllCommands.shortTitle": "Palette dei comandi",

	"command.file.newUntitledFile": "Nuovo file",
	"command.file.save": "Salva",
	"command.file.saveAs": "Salva con nome...",
	"command.file.saveAll": "Salva tutto",
	"command.file.open": "Apri...",
	"command.file.reload": "Ricarica da disco",
	"command.file.reloadWithAutoDetect": "Ricarica con rilevamento automatico formato",
	"command.file.copyFilePath": "Copia percorso del file attivo",
	"command.file.copyFilePath.shortTitle": "Copia percorso",

	"command.view.closeEditor": "Chiudi editor",
	"command.view.closeEditor.shortTitle": "Chiudi",
	"command.view.closeAllEditors": "Chiudi tutti gli editor",
	"command.view.closeAllEditors.shortTitle": "Chiudi tutti",
	"command.view.closeOtherEditors": "Chiudi altri editor",
	"command.view.closeOtherEditors.shortTitle": "Chiudi altri",
	"command.view.focusNextEditor": "Focus editor successivo",
	"command.view.focusPreviousEditor": "Focus editor precedente",

	"command.help.reportIssue": "Segnala problema",

	"menu.file": "File",
	"menu.edit": "Modifica",
	"menu.grid": "Griglia",
	"menu.view": "Visualizza",
	"menu.window": "Finestra",
	"menu.help": "Aiuto",

	"menu.copyAs": "Copia come",
	"menu.pasteAs": "Incolla come",
	"menu.headers": "Intestazioni",
	"menu.fill": "Riempi",
	"menu.grid.sort": "Ordina",
	"menu.transform": "Trasforma",
	"menu.setHeaderRows1": "Imposta 1 riga di intestazione",
	"menu.setHeaderRows2": "Imposta 2 righe di intestazione",
	"menu.setHeaderColumns1": "Imposta 1 colonna di intestazione",
	"menu.setHeaderColumns2": "Imposta 2 colonne di intestazione"
} 
```

## `bundle.l10n.it.json`

`bundle.l10n.*.json` is for the UI texts.
This is AI-generated Italian translation for `bundle.l10n.it.json`.
The keys are the English original.

`bundle.l10n.it.json`

```bundle.l10n.it.json
{
	"Cancel": "Annulla",
	"Restart": "Riavvia",
	"Close": "Chiudi",

	"CSV File Format": "Formato File CSV",
	"Reopen": "Riapri",
	"Apply": "Applica",
	"Encoding": "Codifica",
	"Delimiter": "Delimitatore",
	"Comma": "Virgola",
	"Tab": "Tabulazione",
	"None": "Nessuno",
	"Semicolon": "Punto e virgola",
	"Space": "Spazio",
	"Quote": "Virgolette",
	"Quote when": "Usa virgolette quando",
	"Always": "Sempre",
	"Never": "Mai",
	"Needed": "Necessario",
	"End of line": "Fine riga",
	"Insert final newline": "Inserisci nuova riga alla fine",
	"Allow different column count in each row": "Permetti numero diverso di colonne per riga",

	"Delimiter={delimiter}, Quote={quote}": "Delimitatore={delimiter}, Virgolette={quote}",
	"When needed": "Quando necessario",

	"Select Display Language": "Seleziona lingua di visualizzazione",
	"Restart {appName} to switch to {language}?": "Riavviare {appName} per passare a {language}?",
	"To change the display language to {language}, {appName} needs to restart.": "Per cambiare la lingua di visualizzazione in {language}, {appName} deve essere riavviato.",

	"Sort": "Ordina",
	"Sort Target": "Destinazione ordinamento",
	"All Rows": "Tutte le righe",
	"Selected Rows": "Righe selezionate",
	"Selected Cells": "Celle selezionate",
	"Select multiple rows to sort by selected rows.": "Seleziona più righe per ordinare per righe selezionate.",
	"Select multiple cells to sort by selected cells.": "Seleziona più celle per ordinare per celle selezionate.",
	"Multiple ranges are not supported for sorting by selected cells.": "Gli intervalli multipli non sono supportati per l'ordinamento per celle selezionate.",
	"A to Z": "Da A a Z",
	"Z to A": "Da Z a A",
	"Smallest to Largest": "Dal più piccolo al più grande",
	"Largest to Smallest": "Dal più grande al più piccolo",
	"Oldest to Newest": "Dal più vecchio al più recente",
	"Newest to Oldest": "Dal più recente al più vecchio",
	"Shortest to Longest": "Dal più corto al più lungo",
	"Longest to Shortest": "Dal più lungo al più corto",
	"Blanks First": "Vuoti all'inizio",
	"Blanks Last": "Vuoti alla fine",
	"Column {0}": "Colonna {0}",

	"Match Case": "Maiuscole/minuscole",
	"Use Regular Expression": "Usa espressione regolare",
	"Match Whole Cell": "Confronta intera cella",
	"Find": "Trova",
	"Replace": "Sostituisci",
	"Find Next": "Trova successivo",
	"Find Previous": "Trova precedente",
	"Search in Selection": "Cerca nella selezione",
	"Find in Vertical Direction": "Cerca in direzione verticale",
	"Toggle Replace": "Attiva/disattiva sostituzione",
	"Preserve Case": "Mantieni maiuscole/minuscole",
	"Replace All": "Sostituisci tutto",
	"No results": "Nessun risultato",

	"Do you want to save the changes you made to {0}?": "Vuoi salvare le modifiche apportate a {0}?",
	"Your changes will be lost if you don't save them.": "Le modifiche andranno perse se non le salvi.",
	"Do you want to save the changes to the following {0} files?": "Vuoi salvare le modifiche ai seguenti {0} file?",
	"Save": "Salva",
	"Save All": "Salva tutto",
	"Don't Save": "Non salvare",
	"Failed to save file: {0}": "Impossibile salvare il file: {0}",

	"You are not connected to the internet.": "Non sei connesso a Internet.",
	"You already have the latest version.": "Hai già la versione più recente.",
	"An error occurred while checking for updates: {0}": "Si è verificato un errore durante il controllo degli aggiornamenti: {0}",
	"An error occurred while downloading the update: {0}": "Si è verificato un errore durante il download dell'aggiornamento: {0}",
	"A new version of {appName} is available: {version}": "È disponibile una nuova versione di {appName}: {version}",
	"Install Update": "Installa aggiornamento",
	"Later": "Più tardi",
	"Release Notes": "Note di rilascio",
	"Restart {appName} to install the update.": "Riavvia {appName} per installare l'aggiornamento.",
	"Downloading update...": "Download aggiornamento in corso...",
	"Installing update...": "Installazione aggiornamento in corso...",
	"Restart to Update": "Riavvia per aggiornare",
	"Update on Next Start": "Aggiorna al prossimo avvio",

	"[row][:column]": "[riga][:colonna]",
	"Press 'Enter' to go to row {0}, column {1}": "Premi 'Invio' per andare alla riga {0}, colonna {1}",
	"Press 'Enter' to go to row {0}": "Premi 'Invio' per andare alla riga {0}",
	"Press 'Enter' to go to column {0}": "Premi 'Invio' per andare alla colonna {0}",
	"Type a row number between 1 and {0}": "Inserisci un numero di riga tra 1 e {0}",
	"Type a column number between 1 and {0}": "Inserisci un numero di colonna tra 1 e {0}",
	"Invalid number format": "Formato numero non valido",

	"Select the number of header rows": "Seleziona il numero di righe di intestazione",
	"Select the number of header columns": "Seleziona il numero di colonne di intestazione",
	"no headers": "nessuna intestazione"
}
```

Could you please review these Italian translations and let me know if you find any issues or have suggestions for improvements?
Feel free to ask any questions.
Thank you!

By the way, I didn't know you were a native Italian speaker 🇮🇹

**Comment by lzambarda** [comment]
Hey! I had a look at the files you sent me and found a few translations that could be better, but overall the AI did a good job. It just struggled on certain contexts because the direct translation is not necessarily the one that people are used to.

For brevity, I will just include the keys that I would recommend changing for each file. Another note is that the English values are capitalised, while the automatically generated ones are not.

### `package.l10n.it.json`

```json
"Tab": "Tab",
"Quote when": "Utilizzo virgolette",
"Needed": "Quando necessario",
"Insert final newline": "Inserisci a capo finale",
"Restart {appName} to switch to {language}?": "Riavviare {appName} per cambiare lingua in {language}?",
"A to Z": "Dalla A alla Z",
"Z to A": "Dalla Z alla A",
"Smallest to Largest": "In ordine crescente",
"Largest to Smallest": "In ordine decrescente",
"Match Case": "Differenzia maiuscole/minuscole",
"Match Whole Cell": "Confronta l'intero contenuto della cella",
"Search in Selection": "Cerca all'interno della selezione",
```

### `package.nls.it.json`

```json
"command.app.checkForUpdate": "Controlla la disponibilità di aggiornamenti...",
"command.gridEditor.clear": "Cancella contenuti",
"command.gridEditor.unsetHeaderRows": "Rimuovi righe intestazione impostate",
"command.gridEditor.unsetHeaderColumns": "Rimuovi colonne intestazione impostate",
"command.gridEditor.unsetHeaders": "Rimuovi intestazioni impostate",
"command.gridEditor.sort.aToZ": "Ordina dalla A alla Z",
"command.gridEditor.sort.zToA": "Ordina dalla Z alla A",
"command.gridEditor.sort.smallestToLargest": "Ordina in ordine crescente",
"command.gridEditor.sort.largestToSmallest": "Ordina in ordine decrescente",
"command.gridEditor.transform.swapCase": "Inverti maiuscolo/minuscolo",
"command.showAllCommands.shortTitle": "Riquadro comandi",
"command.file.reloadWithAutoDetect": "Ricarica con rilevamento automatico del formato",
"command.file.copyFilePath": "Copia percorso del file corrente",
"command.view.closeAllEditors.shortTitle": "Chiudi tutto",
"menu.setHeaderRows1": "Imposta 1 riga come intestazione",
"menu.setHeaderRows2": "Imposta 2 righe come intestazione",
"menu.setHeaderColumns1": "Imposta 1 colonna come intestazione",
"menu.setHeaderColumns2": "Imposta 2 colonne come intestazione"
```

I hope this is easy for you to merge, if not I can attach the files directly.
Also, if there is a need for some translations to be shorter, I am happy to try and reduce the length!

**Comment by kohii** [comment]
Thank you so much for the detailed review! I'll incorporate these changes into the translations.

> Another note is that the English values are capitalised, while the automatically generated ones are not.

I have one question about this: Looking at VSCode's Italian translations, while English phrases capitalize all words (e.g., "New File"), Italian translations only capitalize the first word (e.g., "Nuovo file").

![CleanShot 2024-11-17 at 16 24 48@2x](https://github.com/user-attachments/assets/2481d4a7-03fc-4dd0-b10b-cde52218d9a6)


Since this pattern is consistent with the AI-generated translations, I think it might be better to keep this style. What do you think?

**Comment by lzambarda** [comment]
Sounds good to me! For context I did refer to the VSCode localisation too for some fields, so this is consistent and looks good to me!

**Comment by kohii** [comment]
Great! I'll include these changes in the next release.
Thank you very much for your help!

**Comment by kohii** [comment]
Italian localisation is released in [v3.0.0-beta.2](https://github.com/kohii/smoothcsv3/releases/tag/v3.0.0-beta.2)!
Thank you for your support!
